src/SprykerEco/Service/ComputopApi/ComputopApiServiceInterface.php

```diff
@@ -39,7 +39,9 @@ public function generateEncryptedMac(ComputopApiRequestTransfer $requestTransfer
 
     /**
      * Specification:
-     * - Generate header transfer by response array
+     * - Generate header transfer by response array.
+     * - Mac must be equal as soon as required otherwise exception will be thrown.
```

Comment:
as I see, `ComputopApiConverterException` is thrown
pleas update this line similar to line below

Reply:
Fixed

Reply:
still incorrect
as I see the same exception class is used for both cases
so it should be `throws {@link ...} if response does not have expected values or %somethig about mac%

Reply:
Fixed

Reply:
exception throwing mentioned at line 43 is duplicating line 44
line 43 should be removed
line 44 could be adjusted if needed (with some test about MAC being equal / equal to somethig?)

Reply:
still not fixed

Reply:
Fixed.